src/sync/index.js

```diff
@@ -95,40 +93,36 @@ async function sync(db) {
   } catch (err) {
     logger.error(`Error GET https://testnet.qtum.org/insight-api/status?q=getInfo: ${err.message}`);
   }
+
   sequentialLoop(
     Math.ceil((currentBlockChainHeight - startBlock) / batchSize), async (loop) => {
       const endBlock = Math.min((startBlock + batchSize) - 1, currentBlockChainHeight);
 
-      await syncTopicCreated(db, startBlock, endBlock, removeHexPrefix);
-      logger.debug('Synced Topics');
-
-      await Promise.all([
-        syncCentralizedOracleCreated(db, startBlock, endBlock, removeHexPrefix),
-        syncDecentralizedOracleCreated(db, startBlock, endBlock, removeHexPrefix, currentBlockTime),
-        syncOracleResultVoted(db, startBlock, endBlock, removeHexPrefix, oraclesNeedBalanceUpdate),
-      ]);
-      logger.debug('Synced Oracles');
-
-      await Promise.all([
-        syncOracleResultSet(db, startBlock, endBlock, removeHexPrefix, oraclesNeedBalanceUpdate),
-        syncFinalResultSet(db, startBlock, endBlock, removeHexPrefix, topicsNeedBalanceUpdate),
-      ]);
-      logger.debug('Synced Result Set');
-
-      const updateBlockPromises = [];
-      for (let i = startBlock; i <= endBlock; i++) {
-        const updateBlockPromise = new Promise(async (resolve) => {
-          await db.Blocks.insert({
-            _id: i,
-            blockNum: i,
-            blockTime: currentBlockTime,
-          });
-          resolve();
-        });
-        updateBlockPromises.push(updateBlockPromise);
+      try {
```

Comment:
dunt think need this one. Errors r thrown in each func separately. As we r logging but ignoring error anyway, no need to catch them here

Reply:
moved the try catch to the function itself and removed encasing try/catch.